app/models/like.rb

```diff
@@ -1,5 +1,14 @@
 class Like < ApplicationRecord
   # Associations
-  belongs_to :author, class_name: 'User'
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   belongs_to :post
+
+  # Methods
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
@HAMMAS-SALEEM Wow, I didn't even think of that. Thank you